tests/ceph_installer/test_cephadm.py

```diff
@@ -49,6 +57,75 @@
 )
 
 
+def get_host_osd_map(cls):
+    out, err = cls.shell(args=["ceph", "osd", "tree"])
```

Comment:
I think there is a provision to get JSON output of `osd tree` using `-f json`, which will be easy to get data.

Reply:
Thanks for this suggestion. I have used it.